app/src/main/java/nyc/c4q/jonathancolon/inContaq/utlities/sms/weekly/WeeklySent.java

```diff
@@ -0,0 +1,70 @@
+package nyc.c4q.jonathancolon.inContaq.utlities.sms.weekly;
+
+import android.os.AsyncTask;
+import android.util.Log;
+import org.joda.time.DateTime;
+import java.util.ArrayList;
+import java.util.TreeMap;
+import nyc.c4q.jonathancolon.inContaq.utlities.sms.Sms;
+
+public class WeeklySent extends AsyncTask<WeeklyTask, Void, TreeMap<Integer, Integer>> {
+
+    private TreeMap<Integer, Integer> weeklyTexts;
+
+    public WeeklySent() {
+    }
+
+    @Override
+    protected void onPreExecute() {
+        Log.i("WeeklySentTask", "Getting Weekly Total Sent...");
+    }
+
+    @Override
+    protected TreeMap<Integer, Integer> doInBackground(WeeklyTask... params) {
+
+        ArrayList<Sms> listSms = params[0].listSms;
+        weeklyTexts = params[0].weeklyTexts;
```

Comment:
this looks like it doesn't belong in the doInBackground...not sure though. why is this being changed in doInBackground